tests/unit/specs/components/common/TmSessionHardware.spec.js

```diff
@@ -6,8 +6,8 @@ import TmSessionHardware from "common/TmSessionHardware"
 const localVue = createLocalVue()
 localVue.use(Vuex)
 localVue.use(Vuelidate)
-localVue.directive(`tooltip`, () => { })
-localVue.directive(`focus`, () => { })
+localVue.directive(`tooltip`, () => {})
```

Comment:
the issue with the clipboard can be solved by adding this: `localVue.directive(`clipboard`, () => {})`

Reply:
or this: `localVue.directive('clipboard', VueClipboard)`

Reply:
Thank you Jordan! I just fixed it :v: